Makefile

```diff
@@ -119,7 +123,31 @@ OCI_REGISTRY := projects.registry.vmware.com/tce
 
 ##### LINTING TARGETS
 .PHONY: lint mdlint shellcheck check yamllint misspell actionlint urllint imagelint
-check: ensure-deps lint mdlint shellcheck yamllint misspell actionlint urllint imagelint
+check:
+	@make_version=$(shell make --version | grep -E "GNU Make [0-9].[0-9]+" | grep -E -o "[0-9].[0-9]+" | cut -d "." -f1); \
+	if [ "$${make_version}" != "3" ]; then \
+		$(MAKE) check-parallel; \
+	else \
+		$(MAKE) check-serial; \
+	fi;
+
+.PHONY: check-serial
+check-serial:
+	@echo " "
+	@echo "***************************************************"
+	@echo "To speed up linting, update to the latest GNU Make."
+	@echo "***************************************************"
+	@echo " "
+	$(MAKE) lint mdlint shellcheck check yamllint misspell actionlint urllint imagelint
```

Comment:
The `--jobs` argument is available with make 3.x, just not `--output-sync`.  Shouldn't we still try to make this concurrent?

Reply:
Yea, the problem is if you have a failure... it's going to be very difficult to try to make sense of what is going on to try and fix something when you have like output from 7ish different jobs going.